.github/workflows/node-integrate.yml

```diff
@@ -96,6 +100,15 @@ jobs:
           key: packages/aws-lambda-sdk/node-modules-v16-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('node/packages/aws-lambda-sdk/package.json') }}
           restore-keys: packages/aws-lambda-sdk/node-modules-v16-${{ runner.os }}-${{ github.ref }}-
 
+      - name: Retrieve go/packages/dev-mode/node_modules from cache
+        if: steps.pathChanges.outputs.devModeExtension == 'true'
+        id: cacheDevModeExtensionNodeModules
+        uses: actions/cache@v2
+        with:
+          path: node/go/packages/dev-mode/node_modules
+          key: go/packages/dev-mode/node-modules-v16-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('node/go/packages/dev-mode/package.json') }}
```

Comment:
Configuration of dev dependencies for all packages is kept at `node/package.json` so we don't need that.

We may just need to install extra deps for test fixtures, as we do other packages

Reply:
Please confirm that i updated this correctly 👍